packages/manager/modules/pci/src/projects/project/notebooks/Notebook.class.js

```diff
@@ -53,8 +56,15 @@ export default class Notebook {
    * @param notebook {Object}: native JS object
    * @param coreConfig {Object}: used to get link info
    */
-  constructor(notebook, coreConfig) {
-    Object.assign(this, notebook);
+  constructor({ createdAt, id, spec, status, updatedAt, user }, coreConfig) {
+    Object.assign(this, {
+      createdAt,
+      id,
+      spec,
+      status,
+      updatedAt,
+      user,
+    });
     this.coreConfig = coreConfig;
```

Comment:
Instead of passing `coreConfig` maybe we can pass just the element we want (language, user)

Reply:
This is actually not required as of now. Not sure how it got into this PR. Removed...